yt_dlp/cookies.py

```diff
@@ -135,7 +155,13 @@ def _extract_firefox_cookies(profile, logger):
         cursor = None
         try:
             cursor = _open_database_copy(cookie_database_path, tmpdir)
-            cursor.execute('SELECT host, name, value, path, expiry, isSecure FROM moz_cookies')
+            if isinstance(container_id, int):
```

Comment:
Don't check for int here and None above. Use the same check in both places

Reply:
done